tests/parquet_test.py

```diff
@@ -423,27 +423,29 @@ def test_segarray_write(self):
             rd_data = ak.read_parquet(f"{tmp_dirname}/int_test*")
             for i in range(3):
                 self.assertListEqual(s[i].to_list(), rd_data[i].to_list())
+        print("Finished Int")
 
         # integer with empty segments
         a = [0, 1, 2]
         c = [15, 21]
         s = ak.SegArray(ak.array([0, 0, len(a), len(a), len(a), len(a)+len(c)]), ak.array(a + c))
         with tempfile.TemporaryDirectory(dir=ParquetTest.par_test_base_tmp) as tmp_dirname:
-            s.to_parquet(f"{tmp_dirname}/int_test")
+            s.to_parquet(f"{tmp_dirname}/int_test_empty")
 
-            rd_data = ak.read_parquet(f"{tmp_dirname}/int_test*")
+            rd_data = ak.read_parquet(f"{tmp_dirname}/int_test_empty*")
             for i in range(6):
                 self.assertListEqual(s[i].to_list(), rd_data[i].to_list())
+        print("Finished Int Empty")
```

Comment:
```suggestion

```

Reply:
Manually removed this when resolving merge conflicts.